### ISSUE TITLE
Did you obtain permission to use ChatGPT with n2c2 datasets?

### ISSUE THREAD

**Issue by alistairewj** [yhydhx/ChatGPT-API#2]
The data use agreement forbids sharing the data in the manner done by your preprint without explicit permission:

> 5. Data User understands and agrees that the Data / Datasets are proprietary and confidential to Partners and agrees that Data User will not disclose, disseminate, or otherwise share the Data / Datasets to or with any other person or entity, including any subcontractor, for any purpose, without the prior written consent of Partners. To the extent Partners agrees in writing to permit such further access, the Data User will ensure that such further recipient of the Data / Datasets agrees in writing to all of the same restrictions, conditions and obligations that apply to Data User with respect to the Data / Datasets, and will make Partners a third-party beneficiary of such agreement.

Did you obtain such permission, and if so, were there any conditions on it?

**Comment by zl-liu** [comment]
Yes, we did. There are no special conditions other than the ones listed out when you apply and sign the dataset usage agreement.

**Comment by alistairewj** [comment]
I'm not sure I understand. The condition of the agreement is that you do not "share the Data / Datasets to or with any other person or entity ... without the prior written consent". So there definitely is a condition that you don't use this data with ChatGPT in the data use agreement you signed, as that would involve sharing the data with another entity (OpenAI).
